testsuite/features/secondary/min_bootstrap_negative.feature

```diff
@@ -71,3 +71,29 @@ Feature: Negative tests for bootstrapping normal minions
     Then I should see a "Standard Error" text
     And I should see "port 11: Connection refused" or "port 11: Network is unreachable" in the stderr textarea
     When I close the modal dialog
+
+  Scenario: Cleanup: bootstrap a SLES minion after negative tests
```

Comment:
Those changes don't seem related to your PR. Is it a leftover or expected ?

Reply:
Indeed, thank you